### PR TITLE
[Doc] Added TOC for config page

### PR DESCRIPTION
**Pull request by rajalokan**
- Adding table of contents for configuration page
- Let me know if leve 2 bullets need to be added to toc?

### CODE REVIEW

File: site/content/docs/user/configuration.md [hunk 1]
```diff
@@ -15,7 +15,9 @@ This guide covers how to configure KIND cluster creation.
 
 We know this is currently a bit lacking right now and will expand it over time - PRs welcome!
 
-<!--TODO: TOC?-->
+1. [Getting Started](#getting-started)
+2. [Cluster-Wide Options](#clusterwide-options)
```
Comment (BenTheElder) [inline site/content/docs/user/configuration.md:19]: I think we should prefer a TOC that won't get stale.

Comment (rajalokan) [reply site/content/docs/user/configuration.md:19]: Got it. Quick look gives me 
- We can introduce `toc` from hugo side using something like [this](https://gohugo.io/content-management/toc/). But I tried that and it breaks our UI. We would need to bring in CSS changes to accommodate this `toc` ( probably something collapsible either in sidebar or at top of the page). 
- Tried looking at something from markdown side. But that would require adding new plugin. 

Both of this is beyond my knowledge level ( don't even want to try my hand in CSS side of things. :smile: ). Let me know if anything simple can be done.

Comment (BenTheElder) [reply site/content/docs/user/configuration.md:19]: I haven't done this with hugo yet either

**Comment by BenTheElder** [comment]
thanks for the PR but i'm going to hold off for a more future proof approach for now, we already have staleness issues with this sort of content 😅

**Pull request closed without merging**